site/src/component/UserReviews/UserReviews.tsx

```diff
@@ -0,0 +1,53 @@
+import axios, { AxiosResponse } from "axios";
+import React, { FC, useEffect, useState } from "react";
+import SubReview from '../../component/Review/SubReview';
+import Button from 'react-bootstrap/Button';
+import { Divider } from 'semantic-ui-react';
+import { ReviewData } from "src/types/types";
+import './UserReviews.scss';
+import { useCookies } from "react-cookie";
+
+const UserReviews: FC = () => {
+    const [reviews, setReviews] = useState<ReviewData[]>([]);
+    const [loaded, setLoaded] = useState<boolean>(false);
+    const [cookies, setCookie] = useCookies(['user']);
+
+    const getUserReviews = async () => {
+        const response: AxiosResponse<ReviewData[]> = await axios.get(`/reviews?userID=${cookies.user.id}`);
+        setReviews(response.data);
+        setLoaded(true);
+    }
+
+    useEffect(() => {
+        getUserReviews();
+    }, []);
+
+    const deleteReview = async (reviewID: string) => {
+        await axios.delete('/reviews', { data: { id: reviewID } });
+        setReviews(reviews.filter(review => review._id !== reviewID));
+    }
+
+    if (!loaded) {
+        return <p>Loading...</p>;
+    } else if (reviews.length === 0) {
+        return <p>No reviews to display at the moment.</p>;
+    } else {
+        return <div className='user-reviews-container'>
+            <h1>Your Reviews</h1>
+            {/* <p>Verifying a review will display the review on top of unverified reviews.</p> */}
+            <p>Deleting a review will remove it permanently.</p>
+            {
+                reviews.map((review, i) => <div key={`user-reviews-${i}`} className='user-reviews'>
+                    <Divider />
+                    <SubReview review={review}></SubReview>
+                    <div className='user-reviews-footer'>
+                        <Button variant='danger' className='mr-3' onClick={() => deleteReview(review._id!)}>Delete</Button>
+                        {/* <Button variant='success' onClick={() => verifyReview(review._id!)}>Verify</Button> */}
```

Comment:
What is the verify review feature?  Do we want to include this in a future PR?

If so, could you add an issue and describe what the feature is, ideas for implementation, and link to this PR?

Reply:
That was a remnant from the admin page code where admins can verify reviews before they are displayed publicly. I should have removed that line instead of commenting it out.